src/libs/_tests/runtime-utils.test.js

```diff
@@ -253,6 +272,106 @@ const cromwellProvisioningDiskUpdatedPd = {
   zone: 'us-central1-a'
 }
 
+const jupyterStopped = {
```

Comment:
if these mock data consts are only used in one test, I would move them to be in the // ARRANGE section of that test.  If one or more of them are used in more then one test, I would move them to the top of the inside of the appropriate describe arrow function block.  Module scope should be the last resort.